src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    if (style.indexOf(':') === -1) {
+      return acc;
+    }
+
+    const [property, value] = style.split(':')
```

Comment:
When splitting the style into property and value, it's a good practice to limit the number of splits to 2, in case the value itself contains a colon. You can achieve this by passing a second argument to the split method: `style.split(':', 2)`.